fsspec/registry.py

```diff
@@ -154,6 +154,10 @@ def register_implementation(name, cls, clobber=False, errtxt=None):
         "class": "fsspec.implementations.git.GitFileSystem",
         "err": "Install pygit2 to browse local git repos",
     },
+    "smb": {
+        "class": "fsspec.implementations.smb.SMBFileSystem",
+        "err": 'SMBFileSystem requires "smbprotocol[kerberos]" to be installed',
```

Comment:
I agree that it would be better not to require kerberos generally, and catch the auth error when connecting if the particular connection does need it.

Reply:
I hope that the message got better after the last commits.
Can check the changed `err`, please?

Reply:
Adressed in 5391607